index.html

```diff
@@ -28,56 +68,57 @@ <h1 class="ui inverted header">
         {% endif %}
       {% endfor %}
     </p>
-		<a class="ui big orange basic button" href="#type">Découvrez comment&nbsp;&nbsp;⬇︎</a>
-		<a class="ui big teal button" href="{{ site.baseurl }}/api">Explorez les {{ site.api | size }} API déjà disponibles&nbsp;&nbsp;➤</a>
   </div>
-
 </section>
 
-<section id="type">
-  <h2 class="ui divider horizontal">Les Différentes API</h2>
-  <div class="ui items">
+<section id="type" class="ui container">
+  <ul class="ui fluid grid">
     {% for type in site.api_type %}
-    <div class="item">
-      <div class="ui small image">
-        <img src="{{ site.baseurl }}/img/logo.svg" alt="logo">
-      </div>
-      <div class="content">
-        <h3 class="header">{{ type.name }} &nbsp;&nbsp; <a class="ui button" href="{{ site.baseurl }}/api/{{ type.id }}">➤</a></h3>
-        <div class="description">
-          <p>
-            {{ type.description }}
-          </p>
-        </div>
-      </div>
+    <li class="eight wide column">
+      <h3 class="header">{{ type.name }}</h3>
+      <p>
+        {{ type.description }}
+      </p>
+      {% assign api_type_size = site.api | where:"type", type.id |size %}
 
-    </div>
-  	{% endfor %}
-  </div>
+      <p class="discoverAPI">
+        <a href="{{ site.baseurl }}/api/{{ type.id }}">
+          {% if api_type_size == 0 %}
+            Aucune API n'est encore disponible
+          {% elsif api_type_size == 1 %}
+            Explorer la première API
+          {% else %}
+            Explorer ces {{ api_type_size }} API
+          {% endif %}
+        </a>
+      </p>
+    </li>
+    {% endfor %}
+  </ul>
 </section>
 
 <section id="services">
   <h2 class="ui divider horizontal">Ils l'utilisent déjà</h2>
   <div class="ui two stackable cards">
-    {% for service in site.services %}
+    {% for service in page.services %}
     <div class="ui card">
       <div class="image">
-        <img src="{{ "/img/" | append: service.screenshot | append: ".jpg" | prepend: site.baseurl }}" alt="{{ service.name }} screenshot">
+        <img src="{{ "/img/" | append: service.screenshot | prepend: site.baseurl }}" alt="{{ service.name }} screenshot">
```

Comment:
Be consistent with string construction: either use interpolation as in line 85, or use `append`/`prepend` couples.